packages/expo-blob/android/src/main/java/expo/modules/blob/BlobModule.kt

```diff
@@ -0,0 +1,50 @@
+package expo.modules.blob
+
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
+import kotlin.math.max
+import kotlin.math.min
+
+class BlobModule : Module() {
+  override fun definition() = ModuleDefinition {
+    Name("ExpoBlob")
+
+    Class(Blob::class) {
+      Constructor { blobParts: List<BlobPart>?, options: BlobOptionsBag? ->
+        makeBlob(blobParts, options ?: BlobOptionsBag())
```

Comment:
Options already has default value in `makeBlob`, no need for `?: BlobOptionsBag()`